x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/middleware.test.ts

```diff
@@ -307,7 +353,14 @@ describe('middleware', () => {
       await spyMiddleware.waitForAction('trustedAppDeletionSubmissionResourceStateChanged');
       await spyMiddleware.waitForAction('trustedAppsListResourceStateChanged');
 
-      expect(store.getState()).toStrictEqual({ ...testStartState, listView: listViewNew });
+      expect(store.getState()).toStrictEqual({
+        ...testStartState,
+        entriesExist: {
```

Comment:
Yeah - you right. I can do better here. Will create two functions with the needed structure and will reuse that to across all of these

Reply:
This is not specific to you. I don't expect you to refactor these tests in this PR.

The end goal would be not assert whole objects, just the property or two that actually matter. So that we don't have to insert new fields to so many tests, that aren't testing that particular addition